src/test/regress/expected/index_including.out

```diff
@@ -415,17 +415,21 @@ DROP TABLE tbl;
  * 10. Test coverage for names stored as cstrings in indexes
  */
 CREATE TABLE nametbl (c1 int, c2 name, c3 float);
+NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'c1' as the Greenplum Database data distribution key for this table.
+HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
```

Comment:
These lines are unnecessary in this file, the test passes successfully without them.